packages/frameworks/frameworks.json

```diff
@@ -309,6 +309,33 @@
       }
     }
   },
+  {
+    "name": "Storybook",
+    "slug": "storybook",
+    "demo": "https://now-storybook-one.now.sh/",
+    "tagline": "Build bulletproof UI components faster.",
+    "description": "A static deployment of Storybook rendering your components.",
+    "website": "https://storybook.js.org/",
+    "detectors": {
+      "every": [
+        {
+          "path": "package.json",
+          "matchContent": "\"scripts\":\\s*{[^}]*build-storybook\\s*\".+?\"[^}]*}"
+        }
+      ]
+    },
+    "settings": {
+      "buildCommand": {
+        "placeholder": "npm run build-storybook"
+      },
+      "devCommand": {
+        "value": "npm run storybook"
+      },
+      "outputDirectory": {
+        "value": "storybook-static"
```

Comment:
Is `storybook-static` documented somewhere?

[These docs](https://storybook.js.org/docs/basics/exporting-storybook/) make it sounds like the output directory is `.out`?

Reply:
That's manually added with the `-o` (output) flag. The default doesn't seem to be documented, which is whack, but it's defined here: https://github.com/storybookjs/storybook/blob/08cb4a9a03703c0c9b1966e554feac07f272a791/lib/cli/src/generate.js#L73